content/blog/2021/04/2021-04-07-contributhon-participants.adoc

```diff
@@ -0,0 +1,78 @@
+---
+:layout: post
+:title: "SheCodeAfrica Contributhon participants"
+:tags:
+- events
+- community
+:author: markewaite
+:sig: docs
+opengraph:
+  image: /images/post-images/2020-04-contributhon-participants/shecodeafrica-participants-opengraph.png
+---
+
+The link:https://sites.google.com/shecodeafrica.org/contributhon[SheCodeAfrica Contributhon] started April 1, 2021 with participants from Nigeria, Kenya, and Cameroon.
+Twice weekly mentoring sessions are listed in the link:/events[events calendar].
+We've already received the first pull request to improve the Pipeline documentation of key plugins.
+We're looking forward to more pull requests and more improvements throughout April, 2021.
+
+We'd like to introduce our participants so that you recognize them and can welcome them during code review and online chat.
+
+== Onyinye Ezike
+
+image:/images/post-images/2020-04-contributhon-participants/onyinye-ezike.jpg[Onyinye Ezike, width=400]
+
+Onyinye is based in Lagos, Nigeria.
+She's a junior fullstack software developer who is very passionate about learning.
+She uses Angular, React, Nodejs, and Spring Boot.
+She has spent the last two years building up herself in software development and she's hoping to become a world-class software developer.
+
+You'll recognize her contributions as link:https://github.com/Onyimatics[`Onyimatics`] on GitHub and on the link:https://issues.jenkins.io/issues/?jql=creator%20%3D%20%20onyimatics[Jenkins issue tracker].
```

Comment:
Not sure that "issue creator" filter is the best one. Linking the user profile might be the best option, but it won't work without login